src/components/MenuItemWithTopDescription.js

```diff
@@ -20,4 +21,10 @@ function MenuItemWithTopDescription(props) {
 MenuItemWithTopDescription.propTypes = propTypes;
 MenuItemWithTopDescription.displayName = 'MenuItemWithTopDescription';
 
-export default MenuItemWithTopDescription;
+export default React.forwardRef((props, ref) => (
+    <MenuItemWithTopDescription
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...props}
+        forwardRef={ref}
```

Comment:
Sorry I missed this. Just a simple change though, the prop name should `forwardedRef`

Reply:
done!